pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -33,7 +34,8 @@ const (
 	mountPoint          = types.PersistDir
 	defaultImgVault     = types.PersistDir + "/img"
 	defaultCfgVault     = types.PersistDir + "/config"
-	keyDir              = "/TmpVaultDir"
+	keyDir              = "/TmpVaultDir1"
+	oldKeyDir           = "/TmpVaultDir2"
```

Comment:
Isn't the old one /TmpVaultDir?

Reply:
They are just temporary working directories(just within the scope of running a fscrypt command), so the names are arbitrary.  In any case, I have changed the names to oldKeyDir:/TmpVaultDir1 and keyDir:/TmpVaultDir2

Reply:
OK